examples/plotting/aspect.py

```diff
@@ -1,3 +1,11 @@
+''' An example demonstrating how to plot a circle within a rectangle in different aspect scales.
```

Comment:
Suggest some tweak here, the main point to get across is that circles are always circles -- in pixels -- even if the aspect ratio is not 1-1 

```suggestion
''' This example demonstrates how a circle with a data-space radius appears when plotted
with different aspect scales specified.
```

Reply:
Done.